tests/models/rl/integration/test_actor_critic_models.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+from unittest import TestCase
+
+import pytorch_lightning as pl
+
+from pl_bolts.models.rl.advantage_actor_critic_model import AdvantageActorCritic
+
+
+class TestActorCriticModels(TestCase):
```

Comment:
Could you use test functions instead of classes?

> pls do not deprecated unitest, rather a simple test functions

https://github.com/PyTorchLightning/lightning-bolts/pull/453/files#r615373664

Reply:
sure! I'll look into that. Thanks for the update.

Reply:
Done. Also I passed a lot more tests (which is kind of strange).